docs/installation/environment-variables.md

```diff
@@ -142,87 +142,10 @@ Configure the following environment variable to enable MicroPowerManager to send
 Certain MicroPowerManager plugins require configuration before they can be used.
 Find below a reference of configurations which are required if the corresponding plugin is used.
 
-#### Airtel
-
-For detailed information see [Airtel Developer Documentation](https://developers.airtel.africa/developer).
-
-| Environment Variable | Default                            | Description                                 |
-| -------------------- | ---------------------------------- | ------------------------------------------- |
-| `AIRTEL_REQUEST_URL` | **Required** (when plugin is used) | The Airtel service URL.                     |
-| `AIRTEL_USERNAME`    | **Required** (when plugin is used) | The Airtel username.                        |
-| `AIRTEL_PASSWORD`    | **Required** (when plugin is used) | The Airtel password.                        |
-| `AIRTEL_IPS`         | `[]`                               | List of IP whitelisted for Airtel services. |
-
-#### Calin STS Meters
-
-For detailed information see [Calin Meter Developer Documentation](https://www.szcalinmeter.com/).
-
-| Environment Variable | Default                            | Description                                              |
-| -------------------- | ---------------------------------- | -------------------------------------------------------- |
-| `CALIN_CLIENT_URL`   | **Required** (when plugin is used) | Calin Meter client URL used for generating STS tokens.   |
-| `CALIN_USER_ID`      | **Required** (when plugin is used) | Calin Meter API username used for generating STS tokens. |
-| `CALIN_KEY`          | **Required** (when plugin is used) | Calin Meter API key used for generating STS tokens.      |
```

Comment:
These seem used

- https://github.com/EnAccess/micropowermanager/blob/main/src/backend/config/services.php#L43-L53
- https://github.com/EnAccess/micropowermanager/blob/main/src/backend/app/ManufacturerApi/CalinReadMeter.php#L29-L35

Please don't remove them

Reply:
CalinReadMeter.php is not using. We have plugins for Calin & Calin Smart meters. The code sample you shared me is legacy code from very old versions of MPM.  So that can be removed.

Reply:
Resolved here: https://github.com/EnAccess/micropowermanager/pull/526